### PR TITLE
Andrew s/neg vt issue

### PR DESCRIPTION
**Pull request by andrew-sotiriou**
#### What does this PR do?

Fixes Neg VT so that it only works with items that are viewable on the page

#### How can this change be manually tested?

local dev only

#### Asana Card

https://app.asana.com/0/1198498632694766/1202301331735392/f

#### Questions or concerns about this change?

n/a

#### Deployment Notes

This PR will need to be deployed first and will need to then update Oxygen repo

---

#### PR Readiness

* [x] Changes have been validated in local dev

#### Code Readiness

* [x] Code follows [AT Media styles and conventions](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/RCwBYjgA4B)
* [x] Code has been [auto-formatted and is free of lint errors](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/BXnBJVigtW)
* [x] Unrelated code changes are not present
* [x] Relevant inline documentation or READMEs have been updated
* [x] Application and browser logs reviewed for runtime warnings and errors
* [x] Correct level of [logging](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/gwTZjrJhOy) added to code

#### Automated Testing

* [x] [Unit tests](https://apartmenttherapy.slite.com/app/channels/R54jSBx0aG/notes/k6Uah8Iquw) are written at the appropriate level given the code’s risk-level
* [x] Existing tests still pass
* [x] No unnecessary test runner output, e.g. debugging or warnings are not present in Jest or Unittest output

#### Other Considerations

* [x] Relevant documentation updated in Slite

Also consider this PR's impact on architecture and security; AMP, Apple News, RSS; SEO, GA, performance and accessibility.

### CODE REVIEW

File: src/Bling.js [hunk 1]
```diff
@@ -551,7 +551,18 @@ class Bling extends Component {
...
         );
-        if (inViewport) {
+
+        const isHidden = () => {
+            const el = ReactDOM.findDOMNode(this);
+            return el.offsetParent === null || el.style.display === "none";
+        };
+
+        if (inViewport && isHidden() === false) {
+            console.log(
```
Comment (Engywook2005) [inline src/Bling.js]: Of course we should remove this log before deployment...

Comment (andrew-sotiriou) [reply src/Bling.js]: yup...i missed that and just removed

**Review by Engywook2005 — COMMENTED** [review]
Other than the log, I think this looks good... just need to take that log out.

**Review by Engywook2005 — APPROVED** [review]

**Comment by Engywook2005** [comment]
Looks good to me and fixes the warnings.

**Pull request merged**